EIPS/eip-165.md

```diff
@@ -224,6 +224,11 @@ contract Homer is ERC165, Simpson {
 
 With three or more supported interfaces (including ERC165 itself as a required supported interface), the mapping approach (in every case) costs less gas than the pure approach (at worst case).
 
+## Version history
+* EIP 1640, finalized 2019-01-23 -- This corrects the noThrowCall test case to use 36 bytes rather than the previous 32 bytes. The previous code was an error that still silently worked in Solidity 0.4.x but which was broken by new behavior introduced in Solidity 0.5.0. This change was discussed at [#1640](https://github.com/ethereum/EIPs/pull/1640).
```

Comment:
This PR is number 1640, but it's not an EIP - it's a PR.

Reply:
Good catch, thank you. Please also change from "Version history" to "Version History"